### DISCUSSION TITLE
PaddleOCR在昇腾910B部署时，安装albumentations时报错：（include/simsimd/types.h:235:10: fatal error: arm_sve.h: No such file or directory）

### DISCUSSION THREAD

**Discussion by zyscodes** [PaddlePaddle/PaddleOCR#14153] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/1eb76ab9-87a2-4056-bc05-25a6d2388d23)
请教大佬们帮忙看看这个是什么问题? 有搜到需要改动simsimd的源码 确保不启用 SVE，但是源码属实改不动啊。
Chatgpt给的回复并没有解决问题：
![image](https://github.com/user-attachments/assets/6b148d8e-b772-4a57-a06e-0088fae14ace)
恭候大佬针对此问题给予指导帮助。

**Comment by zyscodes** [comment, marked as answer]
Oh,shit！问题解决了，大概原因就是在安装albumentations 依赖时，其内部依赖simsimd组件，不指定simsimd版本时会安装它的最新版本,而 目前simsimd最新版本为6.0.0，其中 当你在 ARM64 架构的服务器上安装 Albumentations 时，如果依赖的 SimSIMD 库出现报错，指向 arm_sve.h 文件无法找到（fatal error: arm_sve.h: No such file or directory），说明当前环境可能不支持 SVE (Scalable Vector Extension)，或者相关的开发包没有正确安装。

SVE 是 ARMv8.2 及其更高版本的扩展，用于支持高效的矢量化计算。
So，重点来了，一行命令解决：pip install albumentations simsimd==1.1.2   安装albumentations 组件时安装simsimd最低版本1.1.2.
安装其他依赖完成后成功运行。
![image](https://github.com/user-attachments/assets/b4c4f7b7-90fa-4097-8222-9c558a2c9216)

**Reply by xiaogege1210** [reply]
哥们牛逼，成功解决
